### ISSUE TITLE
Pointless scrollbar appearing on short-enough sidebar

### ISSUE THREAD

**Issue by Gouvernathor** [executablebooks/sphinx-book-theme#732]
![image](https://github.com/executablebooks/sphinx-book-theme/assets/44340603/958603a7-6ee6-41a2-8ba2-5d472bc01b3e)
As you can see, a scrollbar is generated even though the sidebar's contents is short enough not to need one.
Also, scrolling that scrollbar actually works, so there is too much whitespace in that sidebar for no good reason, and that's probably causing the scrollbar to appear in the first place.

**Comment by Gouvernathor** [comment]
This doesn't happen on [the doc of the theme](https://sphinx-book-theme.readthedocs.io/en/stable/#). To fix this issue, you just have to add the following in the generated HTML, inside the `<div id="rt-footer-container">` : `<span class="fa"></span>`
That's how the theme doc does it (with other divs wrapping it and something inside the span of course), and the fix also works using a div as long as its class is "fa".

**Comment by avivajpeyi** [comment]
Hey @Gouvernathor -- did you find a cleaner fix to this?

**Comment by Gouvernathor** [comment]
Nope, still hanging around with that useless scrollbar. I know almost nothing about CSS.

**Comment by pradyunsg** [comment]
Got a link to a page demonstrating the issue?

**Comment by Gouvernathor** [comment]
https://s3.seedboxws.com/ddl/usr00218/Senna/index.html

**Comment by ebolyen** [comment]
Ran into this also. I found that it seems to be the negative top margin (at least on Firefox) which causes the miscalculation on the total height. 

If the CSS was more specific on the margin (`margin-bottom: -1rem`) I suspect the goal of escaping the sidebar padding would be achieved while allowing the flexbox to work normally (so long as you can tolerate the default gap from flex between the footer and injected ads).

**Comment by mikegerber** [comment]
We also ran into this and used the CSS workaround by @ebolyen.

**Comment by sirex** [comment]
Same here, and for me following worked (inspired by @ebolyen):

```css
#rtd-footer-container {
    margin-top: 0px !important;
    margin-bottom: 0px !important;
}
```

For some reason, I also needed to add `margin-bottom`.
